src/utils/serverConfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I updated my PR to remove the config for the KA and just load it into the servers collection. We should do the same here and remove this config

Reply:
Removed the file